client/src/app/shared-components/recipe-cards/recipe-Cards-style.scss

```diff
@@ -48,29 +48,29 @@
   }
   
 }
-.modal-dialog {
-  max-width: 60rem;
-  .modal-content {
-    width: 60rem;
+// .modal-dialog {
```

Comment:
Let's delete this file, if we don't need it anymore (&& will not need it in the future)

Reply:
deleted comments